src/tests/GC/Stress/Framework/ReliabilityFramework.cs

```diff
@@ -943,7 +943,7 @@ private void StartTest(ReliabilityTest test)
 
             Interlocked.Increment(ref _testsRunningCount);
             test.TestStarted();
-            _logger.WriteToInstrumentationLog(_curTestSet, LoggingLevels.TestStarter, String.Format("RF.StartTest, RTs({0}) - Instances of this test: {1} - New Test:{2}, {3} threads",
+            // _logger.WriteToInstrumentationLog(_curTestSet, LoggingLevels.TestStarter, String.Format("RF.StartTest, RTs({0}) - Instances of this test: {1} - New Test:{2}, {3} threads",
```

Comment:
I would comment the entire line altogether here - seems like this is breaking the build.